mopidy_jellyfin/frontend.py

```diff
@@ -196,6 +196,10 @@ def general_command(self, data):
         if command == 'SetVolume':
             volume = data['Arguments'].get('Volume')
             self.core.mixer.set_volume(int(volume))
+        elif command == 'VolumeUp' or command == 'VolumeDown':
+            vol = self.core.mixer.get_volume().get()
+            increment = -1 if command == 'VolumeDown' else 1
```

Comment:
mopidy tracks volume in a range of 0-100.  That's a lot of adjustment range if we're incrementing by 1.  I think it might be better to increment by 5 instead

Reply:
Updated the PR with the increment changed to 5.